airbyte-integrations/connectors/destination-s3-v2/build.gradle

```diff
@@ -37,6 +37,9 @@ dependencies {
     // temporary dependencies so that we can continue running the legacy test suite.
     // eventually we should remove those tests + rely solely on the bulk CDK tests.
     integrationTestLegacyImplementation testFixtures(project(":airbyte-cdk:java:airbyte-cdk:airbyte-cdk-s3-destinations"))
+
+    // TODO this should come from from the cdk plugin + respect the cdk version
+    integrationTestImplementation testFixtures(project(":airbyte-cdk:bulk:toolkits:bulk-cdk-toolkit-load-avro"))
```

Comment:
fyi - submitted https://github.com/airbytehq/airbyte-internal-issues/issues/11245 + assigned it to myself, I want to knock this out before I forget, but don't want to block this PR on it